tests/scenarios/scenarios.ts

```diff
@@ -72,11 +80,17 @@ async function canary(project: Project) {
 }
 
 export function supportMatrix(scenarios: Scenarios) {
-  return scenarios.expand({
-    lts_3_28,
-    lts_4_4,
-    release,
-  });
+  return (
+    scenarios
+      .expand({
+        lts_3_28,
+        lts_4_4,
+        lts_5_12,
+        release,
+      })
+      // exceeding GitHub actions limit of 256
+      .skip('lts_4_4')
```

Comment:
Skipping these to get around `Strategy expansion exceeded 256 results for job 'test'`, see https://github.com/embroider-build/embroider/actions/runs/12254560402